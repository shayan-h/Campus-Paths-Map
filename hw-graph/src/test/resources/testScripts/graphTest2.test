# Tests if a graph can be created and if nodes can be added

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Print the nodes in the graph
ListNodes graph1