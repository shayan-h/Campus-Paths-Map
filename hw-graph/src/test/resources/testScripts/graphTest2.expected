# Tests if a graph can be created and if nodes can be added

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2